content/kubeflow/install.md

```diff
@@ -70,6 +83,9 @@ Validate that GPUs are available:
 
 ```
 kubectl get nodes "-o=custom-columns=NAME:.metadata.name,MEMORY:.status.allocatable.memory,CPU:.status.allocatable.cpu,GPU:.status.allocatable.nvidia\.com/gpu"
+```
+You should see number of GPU's available in your worker nodes
```

Comment:
Do we want to change to CPU for the workshop?

Reply:
Not yet. I'll have more info Nov 1st week